### PR TITLE
Assign entries for voting

### PR DESCRIPTION
**Pull request by jayjay19630**
### Description

This change allows admin to control when they wish to assign / reassign entries for voting. Doing so will delete all existing vote submissions and essentially 'reboot' the assignment so that each entry has a fair chance of being voted on. 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Code quality improvements

### Checklist

- [x] I have tested this code
- [ ] I have updated the documentation

### CODE REVIEW

File: src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx
Comment (RichDom2185) [inline src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx:176]: Thanks for grouping the state logic together using comments. Perhaps it could be better if we can have a separate sub-subcomponent for this toggle instead for better abstraction? I can imagine once we add more configurations into this cell it can become more complex – the `GroundControlConfigureCell` just be focused on the layout, ideally.

File: src/styles/_groundcontrol.scss
Comment (RichDom2185) [inline src/styles/_groundcontrol.scss:109]: On the same vein as the above comment, these classes can be separated into a CSS module with the abstracted component name.

**Review by RichDom2185 — COMMENTED** [review]
Just a minor comment on the refactor (abstraction). Otherwise, looks good, thanks!

Approved (in-principle), but as usual, we wait for the backend PR to pass review before merging both at the same time.

File: src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx
Comment (jayjay19630) [reply src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx:176]: On that note, do you think it might be a good idea to abstract out the other switches as well (i.e. team assessment, is publish voting, etc.?) I know they're pretty small components but they still might clutter up the code once you add more and more stuff to the config panel

Comment (RichDom2185) [reply src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx:176]: I was thinking about it, and I came to the conclusion that we should just abstract as necessary (i.e. only this component, at least for now) – what do you think?

Comment (jayjay19630) [reply src/pages/academy/groundControl/subcomponents/GroundControlConfigureCell.tsx:176]: Okay then, sounds good. I'll refactor just the assign entries portion inside a new file.

**Comment by jayjay19630** [comment]
I've added a new folder called configureControls, which includes a new `AssignEntriesButton.tsx`. If any more components need to be added to the panel, they can be organised inside that configureControls folder.

Also, I've added a new scss module just for these configureControls. Let me know if there are any more issues.

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**
